spec/models/labware_creators/custom_tagged_plate_spec.rb

```diff
@@ -232,23 +236,25 @@ def expect_state_change_creation
           let(:tag_plate_state) { 'exhausted' }
 
           it 'creates a tagged plate' do
+            # This one will be VERY different
```

Comment:
why?

Reply:
I wish I could tell you.  I just moved the existing comment to align with the line it is referring to.